sdk/identity/azure-identity/CMakeLists.txt

```diff
@@ -12,6 +12,14 @@ project(azure-identity LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+# Compile Options
+option(BUILD_SOURCE_DEPS "build source dependencies" OFF)
```

Comment:
Have validated what happen if we set this option ON from root directory?  should we ignore it?  or should we throw and mention that the option is not for global usage?  Or are we going to let each package be generated as from source ?

Reply:
resolved with no comments?